modules/actions/workflows.go

```diff
@@ -89,13 +97,13 @@ func GetEventsFromContent(content []byte) ([]*jobparser.Event, error) {
 	return events, nil
 }
 
-func DetectWorkflows(commit *git.Commit, triggedEvent webhook_module.HookEventType, payload api.Payloader) (map[string][]byte, error) {
+func DetectWorkflows(commit *git.Commit, ref string, triggedEvent webhook_module.HookEventType, payload api.Payloader, isPullRequestTarget bool) ([]*DetectedWorkflow, error) {
```

Comment:
It seems that the `ref` as a parameter  just goes in and out. It's possible to improve it? Maybe filling `Ref` for DetectedWorkflow outsides?

Reply:
`isPullRequestTarget` looks ugly to me.

How about removing it and filtering the returned workflows outside?

Reply:
done in bf9646885775707af2cda1995effe7f019fd6ab3